### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by rsciupider**
-Added footer html structure
-Added styles
-It's only for mobile for now, no responsiveness

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -34,8 +34,8 @@
   <main>
 
   </main>
-  <footer>
-
+  <footer class="footer">
```
Comment (KJachimstal) [inline src/index.html]: I think better solution is move this footer class in your partial html file. Just add styles from class "footer" to class "footer__container".

**Review by KJachimstal — CHANGES_REQUESTED** [review]

File: src/sass/partials/_footer.scss [hunk 2]
```diff
@@ -19,8 +20,14 @@
     }
   }
 
+  &__container hr {
+    border: none;
+    border-top: 1px solid #ccc;
```
Comment (matilis) [inline src/sass/partials/_footer.scss]: Border-color. I think it is $lineColor: #E1E1E1; not #ccc

**Review by matilis — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (rsciupider) [reply src/index.html]: Done

File: src/sass/partials/_footer.scss (hunk 2 above)
Comment (rsciupider) [reply src/sass/partials/_footer.scss]: Done

File: src/partials/footer.html [hunk 3]
```diff
@@ -1,43 +1,47 @@
...
-        </li>
-      </ul>
-    </div>
-    <div class="footer__contact">
-      <span class="footer__contact--text">Feel free to contact</span>
-      <a class="footer__contact--phone" href="tel:+61 (0) 383 766 284">+61 (0) 383 766 284</a>
+    <div class="footer__tablet--primary">
+      <div class="footer__headline">
+        <p class="footer__headline--primary">
+          homemade ice cream <br />
```
Comment (matilis) [inline src/partials/footer.html:6]: Delete <br /> and use max-width

**Pull request closed without merging**